portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,13 +29,21 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that receives user's comment (if any) and returns all users' comments for that session. */
+/** Servlet that receives user's comment (if any) and returns all users' comments. */
 @WebServlet("/submit_comment")
 public class DataServlet extends HttpServlet {
-  private List<String> comments = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.ASCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Please make this a final field that's instantiated along with declaration

Reply:
Ahh... Yes! Thanks :D

Reply:
Hmm i don't see any changes being made?

Reply:
Sorry, I was busy yesterday. 😅 Just made the changes. :D

Reply:
No problem, feel free to reply/resolve only when you've made the change, otherwise I got an email alert and thinking that you've made the change :XD